components/base/Collapsible.tsx

```diff
@@ -78,15 +85,33 @@ function Collapsible({ triggerText, children }: Props) {
 
   const transitionTime = 200;
 
+  // Sets active state on collapsible
+  function handleActive() {
+    if (!active) {
+      setActive(true);
+    } else {
+      setActive(false);
+    }
+  }
+
+  // Handle "Enter" key to open/close collapsible
+  function handleKeyDown(e) {
+    if (e.keyCode === 13) handleActive();
+  }
+
   return (
-    <CollapsibleWrapper>
+    <CollapsibleWrapper
+      tabIndex={0}
+      onKeyDown={(keyPress) => handleKeyDown(keyPress)}
+      onClick={() => handleActive()}
+    >
       <ReactCollapsible
         trigger={closedTrigger}
         triggerWhenOpen={openTrigger}
         transitionTime={transitionTime}
-        tabIndex={0}
+        open={active}
```

Comment:
This sets the open/close state of the Collapsible. Starts out closed, and changes `active` state based on clicks and keypresses.